src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/subcommands/DumpCommand.java

```diff
@@ -0,0 +1,100 @@
+package io.github.thebusybiscuit.slimefun4.core.commands.subcommands;
+
+import io.github.thebusybiscuit.slimefun4.core.commands.SlimefunCommand;
+import io.github.thebusybiscuit.slimefun4.core.commands.SubCommand;
+import io.github.thebusybiscuit.slimefun4.implementation.Slimefun;
+import me.mrCookieSlime.CSCoreLibPlugin.Configuration.Config;
+import me.mrCookieSlime.Slimefun.api.BlockStorage;
+import org.bukkit.Bukkit;
+import org.bukkit.World;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.ConsoleCommandSender;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Collections;
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Supplier;
+
+/**
+ * Provides commands to dump data for triaging
+ *
+ * @author Sefiraat
+ */
+class DumpCommand extends SubCommand {
+
+    private static final String PATH = Slimefun.instance().getDataFolder().getAbsolutePath() + "\\dumps\\";
```

Comment:
This is a very Windows path
```suggestion
    private static final String PATH = Paths.get(Slimefun.instance().getDataFolder().getAbsolutePath(), "dumps");
```